GetStream/stream-chat-swift
# [CIS-1582] Fix `ChatMentionSuggestionView` permanently hiding subviews

### 🔗 Issue Link

CIS-1582

### 🎯 Goal

Fix issue in `ChatMentionSuggestionView` when once hidden subview was never shown again. This happens when users without the `name` are suggested.

### 🛠 Implementation

Manipulate `isHidden` depending on the text that is displayed.

### 🎨 Changes

https://user-images.githubusercontent.com/12818985/154441646-6b103806-cfaf-457d-b342-d0ecbb4a9870.mp4

### 🧪 Testing

Manual testing:
- Run `DemoApp`
- Create a group channel where one user does not have `name`
- Type `@` in the channel multiple times

See that the `name` label is hidden only for the user without `name` and always shown for other.

### ☑️ Checklist
